saas/values.yaml

```diff
@@ -0,0 +1,11 @@
+namespace: baas-system
+deployName: bc-saas
+deployImage: hyperledgerk8s/bc-saas:6b0ed39
+saasIngressName: bc-saas-ingress
+port: 9999
+addr: ""
+ingressClassName: portal-ingress
+ingressDomain: <replaced-ingress-nginx-ip>.nip.io
+saasIngressName: bc-saas-ingress
+serviceName: bc-saas-service
+testContract: go-contract
```

Comment:
Since our saas may have NFT or other service ,we should use `depositoryContract: depository` for `depository service`